packages/fast-jss-utilities/src/color.ts

```diff
@@ -0,0 +1,13 @@
+export function convertHexToRgb(hex: string, alpha?: number): string {
```

Comment:
Part of me thinks that we shouldn't expose the alpha logic in this function. It doesn't really seem part of the scope of converting hex to rgb. 

Also, chroma already does color conversion - which we take a dependency on. Is there a reason we aren't just using chromas utilities?

Reply:
At some point I think there was a conversation that over time we want to reduce our dependency on Chroma where we can for smaller things like this. We can't tree shake it, so if we just wanted this functionality, we are still taking the whole thing.

If that's not a huge concern, I'll just abandon this. 

There is currently an issue on chroma for converting to ES6 to support this, but it's been open for over a year. We could always contribute it in

Reply:
Yea on one hand adding this now reduces the amount of work we'ed need to do later. On the other, it adds code in the interim and moving away from chroma isn't super high priority at the moment. 

I think i'm in favor of just using chroma for now and doing a migration if/when it makes sense, but I'm open to other's thoughts on the matter.

Reply:
I'm fine with this. I'll just close it out.